src/NuGet/DiagnosticSourceListener/build/Microsoft.ApplicationInsights.DiagnosticSourceListener.targets

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="_Ms-AI-DiagnosticSourceListener-EnsureApplicationInsightsConfig" BeforeTargets="PrepareForBuild">
+    <ItemGroup>
+      <_AppInsightsConfigFile Include="@(None)" Condition="'%(Filename)%(Extension)' == 'ApplicationInsights.config'" />
+    </ItemGroup>
+    <Warning Text="Can’t find ApplicationInsights.config. Without this file Application Insights may not work." Condition="@(_AppInsightsConfigFile) == ''" />
```

Comment:
I believe I gave the same feedback there. @karolz-ms can you pls take a look and remove it there as well?